src/app/services/itinerary.service.ts

```diff
@@ -11,15 +11,34 @@ export class ItineraryService {
 
   constructor(private httpClient: HttpClient) { }
 
-  hostURL = 'https://be-dev-yourney.herokuapp.com/';
+  // hostURL = 'http://localhost:3000/';
+  id = 1
+  hostURL = 'https://be-dev-yourney.herokuapp.com';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
  hostURL = "https://be-dev-yourney.herokuapp.com";
```